### DISCUSSION TITLE
Colour of ground

### DISCUSSION THREAD

**Discussion by ilkant** [Stellarium/stellarium#3237] [Q&A] [answered]
When using simple landscape the color of ground is green. I searched setting to it's color from F4 settings. Not found. So, is that ground color defined on landscape? And how to change it to very dark green or black?

**Comment by gzotti** [comment, marked as answer]
The ground color is defined by the ground_color entry in a polygonal landscape description. See User Guide, chapter 7. Color was inspired by some 1990s program. There is no way for the user to change it interactively. Define your own landscape so change it. 

#3180 will introduce a way to dynamically set polygonal landscape color from the map.

**Reply by ilkant** [reply]
Thanks for the good answer!
